README.md

```diff
@@ -22,24 +25,37 @@ do: [ :baseline | baseline
     load: 'default' ].
 ```
 ###Set Up
-Open the Issue Package Browser (Apps -> Issue Package Browser) and select the package you are working on. Enter the following information:
-- **Project**: The name of your github project e.g. ```HPI-SWA-Teaching/SqueakIssueIntegration```
-- **Login Key**: To authenticate you at the Issue Management System you need a OAuth-Token. You can generate one for your GitHub profile [here](https://github.com/settings/tokens) (needed scopes: "public_repo" and (if using private repositories) "repo")
+Open the Issue Package Browser (`Apps` -> `Issue Package Browser`) and select
+the package you are working on. Enter the following information:
+- **Project**: The name of your github project e.g. `HPI-SWA-Teaching/SqueakIssueIntegration`
```

Comment:
Mein Kommentar, den ich auch schon in Telegram geschrieben habe: Ist das Absicht, dass hier das alte Projekt steht? - Ja. Gut, danke. :D